lib/train/transports/local.rb

```diff
@@ -52,6 +57,133 @@ def login_command
       def uri
         'local://'
       end
+
+      class WindowsRunner
+        require 'json'
+        require 'base64'
+        require 'securerandom'
+
+        def initialize
+          @pipe = acquire_pipe
```

Comment:
I would just throw a train exception that can be handled and we have a proper fallback, which would be the `WindowsShellRunner`

Reply:
Agreed! I make it so in the latest commits.

Reply:
So much cleaner now, thank you!